test/JDBC/expected/BABEL-5119-vu-verify.out

```diff
@@ -0,0 +1,186 @@
+-- tsql
+CREATE LOGIN login_babel5119_1 WITH PASSWORD = '12345678'
+CREATE LOGIN login_babel5119_2 WITH PASSWORD = '12345678'
+ALTER authorization on database::BABEL5119_1 TO login_babel5119_1
+GO
+-- terminate-tsql-conn
+
+-- tsql user=login_babel5119_1 password=12345678
+USE master
+GO
+CREATE TABLE BABEL5119_t4(a int)
```

Comment:
why this login has privilege to create objects in master database?
Also do print current_user here.

Reply:
It creates the objects in guest schema of the master database.

Reply:
Added a comment for better clarity in future